doc/sphinx/language/core/records.rst

```diff
@@ -234,13 +339,15 @@ During the definition of the one-constructor inductive definition, all
 the errors of inductive definitions, as described in Section
 :ref:`gallina-inductive-definitions`, may also occur.
 
-.. seealso:: Coercions and records in section :ref:`coercions-classes-as-records` of the chapter devoted to coercions.
+.. seealso:: Coercions and records in section :ref:`coercions-classes-as-records`.
```

Comment:
What is this about? Should it be moved?

Reply:
I moved it to the record definition section.  I didn't want to dig into the details of coercions right now (don't know much about them).